CMakeLists.txt

```diff
@@ -262,7 +267,6 @@ list(APPEND INCLUDE_FILES
     ${PROJECT_SOURCE_DIR}/include/mbgl/platform/thread.hpp
     ${PROJECT_SOURCE_DIR}/include/mbgl/renderer/query.hpp
     ${PROJECT_SOURCE_DIR}/include/mbgl/renderer/renderer.hpp
-    ${PROJECT_SOURCE_DIR}/include/mbgl/renderer/layer_tweaker.hpp
```

Comment:
There are some duplicate or wrong place files in base branch `topic/drawable`. I fixed them on this branch.